tests/dialects/test_clickhouse.py

```diff
@@ -870,3 +870,45 @@ def extract_agg_func(query):
         )
 
         parse_one("foobar(x)").assert_is(exp.Anonymous)
+
+    def test_hash_functions(self):
+        self.validate_all(
```

Comment:
maybe let's just add all of these to the bigquery tests so that they are all in one place

Reply:
Put it to the bigquery tests is ok, but maybe other contributors will confuse about why bigquery tests fail when the code of clickhouse change.  

Nvm, I will do it